packages/manager/modules/pci/src/projects/project/storages/object-storage/users/delete/delete.html

```diff
@@ -0,0 +1,13 @@
+<oui-modal
+    data-heading="{{:: 'pci_projects_project_storages_containers_users_delete_title' | translate }}"
+    data-primary-label="{{:: 'pci_projects_project_storages_containers_users_delete_label' | translate }}"
+    data-primary-action="$ctrl.delete()"
+    data-secondary-label="{{:: 'common_cancel' | translate }}"
```

Comment:
**thought:** We tend to avoid to rely on `common` translation.
Every module should bring its own translation.